### ISSUE TITLE
unpaired image problem

### ISSUE THREAD

**Issue by weizhou1991** [GaParmar/img2img-turbo#21]
Hi there, 

Thank you so much for your wonderful work, however, when I tried your example for "night to day" model with my own images, the results looks like an overexposure picture, I think it is because of the distribution of the training data is different with the pic I used ? Is there any other reason causing this ? I really appreciate for your help !

Best,

**Comment by weizhou1991** [comment]
of courser, here is the input 
![night_test_2](https://github.com/GaParmar/img2img-turbo/assets/110662434/21c4c666-68f3-4095-a0ca-c8172a2c1534)
 and this is the ouput 
![night_test_2](https://github.com/GaParmar/img2img-turbo/assets/110662434/45f91c32-492f-46b1-a160-262834a974bb)

**Comment by GaParmar** [comment]
Yeah these images look very out of distribution for my pre-trained night-to-day model. 
The pretrained model is trained on BDD images. 
I suspect you will achieve better results if you train/finetune on your dataset directly!
